ci/test_integration.sh

```diff
@@ -20,6 +20,10 @@ echo "==== ASR (backend=pytorch, lm=TransformerLM) ==="
 ./run.sh --python "${python}" --stage 5 --lm-config conf/lm_transformer.yaml --decode-config "$(change_yaml.py conf/decode.yaml -a api=v2)"
 echo "==== ASR (backend=pytorch, dtype=float64) ==="
 ./run.sh --python "${python}" --stage 3 --train-config "$(change_yaml.py conf/train.yaml -a train-dtype=float64)" --decode-config "$(change_yaml.py conf/decode.yaml -a api=v2 -a dtype=float64)"
+if [ "${TEST_QUANTIZATION:-true}" ]; then
```

Comment:
```suggestion
if "${TEST_QUANTIZATION:-true}"; then
```
sorry, my suggest was wrong